src/main/java/software/amazon/cloudformation/proxy/AmazonWebServicesClientProxy.java

```diff
@@ -110,6 +122,70 @@ public ClientT client() {
         };
     }
 
+    public <ClientT, ModelT, CallbackT extends StdCallbackContext>
+        Initiator<ClientT, ModelT, CallbackT>
+        newInitiator(@Nonnull Supplier<ClientT> client, final ModelT model, final CallbackT context) {
+        return newInitiator(newProxy(client), model, context);
+    }
+
+    @Override
+    public <ClientT, ModelT, CallbackT extends StdCallbackContext>
+        Initiator<ClientT, ModelT, CallbackT>
+        newInitiator(final ProxyClient<ClientT> client, final ModelT model, final CallbackT context) {
```

Comment:
so the callGraph is moved to overload of `.initiate` ln:160? There would be cool to have options for `initiate`:
* The traditional approach `initiate(String callGraph, ProxyClient<ClientT> client, ModelT model, CallbackT cxt)`
* The new one is `initiate(String callGraph)`
* maybe it would be useful to generate the value and dont let the developer to worry about it? something like `initiate()` which uses generated value (like `"handler::execution-context"` which concats the requests and responses - still be meaningful). Handlers dont interfere with each other, so I assume there wont be any issues.

Reply:
We could add self generation later. Right now developers control which makes debugging easy and potentially access things from callback context easy.